server/protocols/webadmin/webadmin-mailbox-deleted-message-vault/src/main/java/org/apache/james/webadmin/vault/routes/DeletedMessagesVaultDeleteTask.java

```diff
@@ -34,7 +34,7 @@
 
 public class DeletedMessagesVaultDeleteTask implements Task {
 
-    static final TaskType TYPE = TaskType.of("deletedMessages/delete");
+    public static final TaskType TYPE = TaskType.of("deletedMessages/delete");
```

Comment:
by using this constant in tests, we make it easy to change the value. Is it a property we want? I would rather say we want to make it hard to break the existing endpoint path

Reply:
I hesitated, but you are right, I will make the change

Reply:
fixed